### ISSUE TITLE
Documentation about ScoutSuite report

### ISSUE THREAD

**Issue by mkarnscgi** [nccgroup/ScoutSuite#1324]
Hello - I am looking for documentation that explains ScoutSuite reporting parameters (resources, rules, findings, checks) mean. Do you have any documentation or know where some is that explains the ScoutSuite reports?

**Comment by p3rzanov** [comment]
Hi,
Did you tried open the 'Wiki' page?

**Comment by mkarnscgi** [comment]
> Hi,
> Did you tried open the 'Wiki' page?

Yes, the wiki page showed a screen shot of what the report looks like but did not explain what things mean
